frontend/src/metabase/core/components/SelectButton/SelectButton.styled.tsx

```diff
@@ -17,11 +18,11 @@ const getColor = ({ hasValue, highlighted }: SelectButtonRootProps) => {
 };
 
 export const SelectButtonRoot = styled.button<SelectButtonRootProps>`
+  ${inputPadding}
```

Comment:
I'm almost certain we can use `inputFocusOutline` here too and remove the duplicate css properties